packages/service-library/tests/fastapi/conftest.py

```diff
@@ -28,3 +33,37 @@ def _get_data(x: PositiveFloat, y: int = Query(..., gt=3, lt=4)):
     _app.include_router(api_router)
 
     return _app
+
+
+@pytest.fixture(params=["", "/base-path", "/nested/path"])
+def router_prefix(request: FixtureRequest) -> str:
+    return request.param
+
+
+@pytest.fixture
+async def bg_task_app(router_prefix: str) -> AsyncIterable[FastAPI]:
+    app = FastAPI()
+
+    long_running.server.setup(app, router_prefix=router_prefix)
+    yield app
+
+
+@pytest.fixture(scope="function")
+async def async_client(bg_task_app: FastAPI) -> AsyncIterable[AsyncClient]:
```

Comment:
Please check my comment on the ``_fastapi.in``!!!

Reply:
replied above